src/lib/pipeline.ml

```diff
@@ -324,9 +328,11 @@ module Construct = struct
        make_target }
       bam_pair
 
-end
+  let rna_hla_typer fastq_sample = Rna_hla_typer fastq_sample
```

Comment:
`gunzip` is not a "kind" of tool, it an actual unix executable associated with a well defined format. If we want to start using `bzip2` we will call it `bzip2`.
 

My concern is that, as a user, I don't want to read `Construct.rna_hla_type` in any pipeline; and I don't want my pipelines to to fail (or do something different) in the future when Biokepi authors remove the function or, worse, change its meaning.